fleetspeak/src/client/clienttestutils/clienttestutils_unix.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build linux darwin
+
+package clienttestutils
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	fspb "github.com/google/fleetspeak/fleetspeak/src/common/proto/fleetspeak"
+)
+
+func writeServiceConfigImpl(content []byte, dirpath, filename string, cfg *fspb.SignedClientServiceConfig) error {
+	if err := ioutil.WriteFile(filepath.Join(dirpath, filename), content, 0644); err != nil {
+		return fmt.Errorf("Unable to write signed service config[%s -> %s]: %v", dirpath, filename, err)
```

Comment:
Error message should start in lowercase.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Done.